runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -340,7 +340,7 @@ stackSlotIteratorForScavenge(J9JavaVM *javaVM, J9Object **slotPtr, void *localDa
 }
 
 bool
-MM_ScavengerDelegate::doContinuationObject(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, MM_ScavengeScanReason reason)
+MM_ScavengerDelegate::doContinuationNativeSlots(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, MM_ScavengeScanReason reason)
```

Comment:
This should probably be 'scan' (or maybe 'scavenge' - there is mixed bag of these 2 terms already). 'Do' makes the least sense.